Gruntfile.js

```diff
@@ -11,7 +11,7 @@ module.exports = function(grunt) {
         trailing: true,
         undef: true,
         unused: true,
-        indent: 2,
+        // indent: 2,
```

Comment:
you'll want to leave this as indent of 2 on the next assignment - you can set Sublime to use 2 spaces, instead of 4 (or tab spacing). Ask me if need help.